ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/less/productdetails.less

```diff
@@ -175,4 +175,37 @@
 			margin-bottom: 30px;
 		}
 	}
+}
+
+
+.fa-twitter {
+ background: white;
+ color: #55ACEE;
+ }
+ .fa-facebook-official{
```

Comment:
@uditnaagar777 plz remove css related to share component from pdp less file and include that in clientlibs of share component

Reply:
This is done. Thanks for the review